content/blog/configuring-azure-ad-with-greenlake-cloud-platform-and-aruba-central.md

```diff
@@ -0,0 +1,236 @@
+
+* If everything is working correctly, you should have logged into GLCP and Aruba Central is an option to Launch.
+
+## Using Azure AD MFA
+* By default, Azure AD enables MFA. However, for testing and demos, it's much easier to disable MFA on your accounts. To disable MFA, please see the following documentation: [What are security defaults](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/concept-fundamentals-security-defaults)
+
+## Troubleshooting
+* There's a useful 3rd party browser tool called: SAML Tracer
+* This tool will allow you to verify the attributes you're sending to Central.
+* It can be useful when configuratin SAML with multiple Central accounts or domains
+* SAML Tracer
+[Chrome](https://chrome.google.com/webstore/detail/saml-tracer/mpdajninpobndbfcldcmbpnnbhibjmch?hl=en)
+[FireFox](https://addons.mozilla.org/en-US/firefox/addon/saml-tracer/)
+
+![Image](/img/0b085a5aef05404e9ecdf52cb9088feb/firefox-saml-tracer.png)
+
+
+
+
+
+## Appendix: Generating the `hpe_ccs_attribute`
+
+The `hpe_ccs_attribute` is used to determine your GLCP account.  The format for the `hpe_ccs_attribute` is as follows:
+
+
+<img src="/img/0b085a5aef05404e9ecdf52cb9088feb/hpe_ccs_attribute-img1.png" alt="hpe_ccs_attribute-img1" height="75%" width="75%">
+
+
+An Example `hpe_ccs_attribute` for a single GLCP and Aruba Central account would be:
+
+```
+version_1#2fd5f97acbc211ecadc006baf610dd36:00000000-0000-0000-0000-000000000000:Account Administrator:ALL_SCOPES:683da368-66cb-4ee7-90a9-ec1964768092:Aruba Central Administrator:ALL_SCOPES
+```
+
+or
+
+
+```
+version_1#5b0ec0e8b4f411eca432ba72799953ac:00000000-0000-0000-0000-000000000000:Account Administrator:ALL_SCOPES:683da368-66cb-4ee7-90a9-ec1964768092:Aruba Central Administrator:ALL_SCOPES#5b0ec0e8b4f411eca432ba72799953ac:00000000-0000-0000-0000-000000000000:Account Administrator:ALL_SCOPES
+```
+
+
+If you're a Managed Service Provider (MSP), then the `hpe_ccs_attribute` for Administrator rights to GLCP and Aruba Central for all customer tenant accounts:
+
+```
+version_1#d951f8c8c67711eca2cf9efb55836a4d:00000000-0000-0000-0000-000000000000:Account Administrator|TENANT|:ALL_SCOPES:00000000-0000-0000-0000-000000000000:Account Administrator|MSP|:ALL_SCOPES:683da368-66cb-4ee7-90a9-ec1964768092:Aruba Central Administrator|TENANT| : ALL_SCOPES:683da368-66cb-4ee7-90a9-ec1964768092:Aruba Central Administrator|MSP| : ALL_SCOPES
+```
+
+
+The `hpe_ccs_attribute` string for a tenant under a MSP account, would be below. However, you **must** have the SAML domain configuration configured for that tenant account using the **same** setting as the MSP account. To say it another way, you **must** go through this configuration for each tenant account under the MSP.
+
+```
+version_1#f9ee1cdecc1611ecb00e9e24ed17d2a7:00000000-0000-0000-0000-000000000000:Observer|TENANT| :ALL_SCOPES:683da368-66cb-4ee7-90a9-ec1964768092:Aruba Central Administrator|TENANT| :ALL_SCOPES
+```
+
+
+
+
```

Comment:
If you could add a note at the very end as a summary of the post that includes some sort of call to action, that would be great. Suggesting readers connect with you via some sort of twitter link/handle, email, or whatever works well. You can also just resort to what we tend to use a lot: “Be sure to come back to the [HPE Developer Community blog](https://developer.hpe.com/blog) for more articles on this and other interesting subjects.”

Reply:
Additionally, you should probably make some sort of statement like the following:'
"I hope this blog post helped you understand how to..." (and then basically summarize the benefit of going through this process... maybe something like "understand how to take advantage of SSO using Azure AD and Aruba Central with the HPE GreenLake Cloud platform".